core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java

```diff
@@ -17,4 +17,26 @@ public void testResolve_knownTokens() {
         assertThat(Long.parseLong(uniqueId) > 1, is(true));
 
     }
+    
+    @Test
+    public void testResolveSystemProperty_PROPERTY_FOUND() {
+    	
+    	String exampleInputString = "zerocode-tokentest: ${SYSTEM.PROPERTY:java.vendor}"; // java.vendor = Oracle Corporation
+    	String resolvedString = resolveKnownTokens(exampleInputString);
+    	String resolvedToken = resolvedString.substring("zerocode-tokentest: ".length());
+    	
+    	assertThat(resolvedToken.equals("Oracle Corporation"), is(true));
```

Comment:
I am not sure of putting `Oracle Corporation` here. So instead of java vendor we can use something like `file.encoding=UTF-8` which is more generic.

Reply:
fixed.